### PR TITLE
Website 2.0

### PR DESCRIPTION
**Pull request by kingsleyzissou**
- Version routing seems to be broken (working on it at the moment)
- Server won't start once the website is brought into the Graphback repo

### CODE REVIEW

**Comment by kingsleyzissou** [comment]
Okay the current blog was breaking things.
I have removed it for now

**Comment by wtrocki** [comment]
Verifying this now

File: website/src/components/Features/index.jsx [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    description: (
+      <>
+        {/* Docusaurus lets you focus on your docs, and we&apos;ll do the chores. Go
+        ahead and move your docs into the <code>docs</code> directory. */}
+      </>
+    ),
+  },
+  {
+    title: <>Deploy</>,
+    imageUrl: 'img/rocket.png',
```
Comment (wtrocki) [inline website/src/components/Features/index.jsx:29]: Would it be possible to make it little bit bigger. Taking 8-% of the available space in rectc

File: website/src/components/Terminal/index.jsx [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+export function Terminal() {
+
+  return (
+    <div>
+      <div className={styles.terminalHeader}>
+          <p>Quick start!</p>
+      </div>
+      <div className={styles.terminal}>
+        <span>>&nbsp;&nbsp;</span>
+        <TerminalText phrase={'npx graphback init example'} />
```
Comment (wtrocki) [inline website/src/components/Terminal/index.jsx:94]: `npx graphback-cli init your-app`

File: website/src/components/Hero/index.jsx [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+              <li>Full control over the generated code or runtime framework</li>
+              <li>Manages complexity of database management</li>
+            </ul>
+            <div className={styles.buttons}>
+              <Link
+                className={classnames(
+                  'button button--primary button--lg button--rounded',
+                  styles.getStarted,
+                )}
+                to={useBaseUrl('docs/doc1')}>
```
Comment (wtrocki) [inline website/src/components/Hero/index.jsx:29]: Let's replace that with the getting started link

File: website/src/components/Features/index.jsx (hunk 1 above)
Comment (kingsleyzissou) [reply website/src/components/Features/index.jsx:29]: Still working on that

File: website/src/components/Hero/index.jsx [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+                Get Started
+                </Link>
+            </div>
+          </div>
+          {/* <div className="col col--6 text--left">
```
Comment (wtrocki) [inline website/src/components/Hero/index.jsx:34]: This comment can be removed

File: website/src/components/Terminal/index.jsx (hunk 2 above)
Comment (kingsleyzissou) [reply website/src/components/Terminal/index.jsx:94]: Cool

File: website/src/components/Hero/index.jsx (hunk 4 above)
Comment (kingsleyzissou) [reply website/src/components/Hero/index.jsx:34]: That's the terminal/code component. Was still working on this, but have commented it out for the meantime. Will maybe extract it to a separate component for now so it's not polluting the code

File: website/src/components/Hero/index.jsx (hunk 3 above)
Comment (kingsleyzissou) [reply website/src/components/Hero/index.jsx:29]: Missed that one. Cool

File: website/docusaurus.config.js [hunk 5]
```diff
@@ -0,0 +1,126 @@
...
+            // },
+          ],
+        },
+        {
+          to: 'docs/gettingstarted',
+          label: `v0.12.x`,
+          position: 'right',
+        },
+        {
+          href: 'https://github.com/facebook/docusaurus',
```
Comment (wtrocki) [inline website/docusaurus.config.js:52]: Change this to graphback

File: website/src/components/Features/index.jsx (hunk 1 above)
Comment (wtrocki) [reply website/src/components/Features/index.jsx:29]: No need to release website actually we can still merge it

File: website/docusaurus.config.js (hunk 5 above)
Comment (kingsleyzissou) [reply website/docusaurus.config.js:52]: Oh haven't pushed that yet. I have fixed that already

**Comment by wtrocki** [comment]
Good stuff. I have suggested very small improvements. Once that is done we can merge and publish website.

**Comment by kingsleyzissou** [comment]
Fixing the routing issues still. Almost there

**Comment by wtrocki** [comment]
We have sidebars.json and sidebars.js - seems like duplicate

**Comment by kingsleyzissou** [comment]
PR is still WIP, it was for the layout. Still quite a few fixes that I need to sort out

**Comment by kingsleyzissou** [comment]
Should be good now. Not too sure about the publish command though

**Comment by craicoverflow** [comment]
> Should be good now. Not too sure about the publish command though

`USE_SSH=true yarn publish-gh-pages`

File: website/docusaurus.config.js [hunk 6]
```diff
@@ -0,0 +1,142 @@
...
+module.exports = {
+  title: 'Graphback',
+  tagline: 'GraphQL API for your client and server',
+  url: 'https://graphback.dev',
+  baseUrl: '/',
+  favicon: 'img/favicon.ico',
+  organizationName: 'aerogear', // Usually your GitHub org/user name.
+  projectName: 'graphback', // Usually your repo name.
+  themeConfig: {
+    disableDarkMode: true,
```
Comment (craicoverflow) [inline website/docusaurus.config.js:13]: is this incompatible with the new design?

File: website/docusaurus.config.js [hunk 7]
```diff
@@ -0,0 +1,142 @@
...
+        },
+        {
+          title: 'Social',
+          items: [
+            // {
+            //   label: 'Blog',
+            //   to: 'blog',
+            // },
+            {
+              label: 'GitHub',
```
Comment (craicoverflow) [inline website/docusaurus.config.js:111]: Should GitHub also fall under community? It is not really social. I guess we don't really have any social channels

File: website/src/components/Terminal/index.jsx [hunk 8]
```diff
@@ -0,0 +1,99 @@
...
+export function Terminal() {
+
+  return (
+    <div>
+      <div className={styles.terminalHeader}>
+          <p>Quick start!</p>
+      </div>
+      <div className={styles.terminal}>
+        <span>>&nbsp;&nbsp;</span>
+        <TerminalText phrase={'npx graphback init your-app'} />
```
Comment (craicoverflow) [inline website/src/components/Terminal/index.jsx:94]:
```suggestion
        <TerminalText phrase={'npx graphback-cli init your-app'} />
```

File: website/docusaurus.config.js (hunk 6 above)
Comment (kingsleyzissou) [reply website/docusaurus.config.js:13]: No, it is compatible, just needs some work in terms of styling. So I'll work on that next, possibly in another pr

**Comment by wtrocki** [comment]
@kingsleyzissou Do you mind publishing new change live?

Always to prod :D

**Comment by wtrocki** [comment]
Since this is live already I think we should.merge.it as new changes to website and docs will be coming

**Pull request merged**